### DISCUSSION TITLE
Just a Sample v1

### DISCUSSION THREAD

**Discussion by BOBONA** [BOBONA/Just-a-Sample#2] [Announcements]
<img src="Assets/plugin_full.png" width="100%" alt="View of full plugin"/>

# Just a Sample

[Your favorite sampler shouldn't be complicated.](https://bobona.github.io/just-a-sample/)

## Overview
Just a Sample is a powerful, _modern_ audio sampler, with a focus on simplicity and ease of use. 
Best of all, it's **free** and **open-source**! I spent a lot of time on this project, and I think it will
be a great addition to your plugin collection. 

Feel free to take a look at the source code and reach out with any questions or suggestions. I took care
to make the code as clean and readable as possible, with a focus on good design practices. That being said, 
I am still new to audio programming.

#### Core features
- Smoothly zoom in to the level of individual samples to 
set bounds as accurately as you need. Waveform drawing is **optimized** for large samples.
- Integrated [Bungee time stretcher](https://github.com/kupix/bungee) allows for
freely modulating time and pitch independently. Try extreme slow-downs (0.01x) for unique sounds.
- Modern navigation controls allow for easy browsing of waveforms with touchpad or mouse.
- A routable FX chain includes reverb, chorus, distortion, and EQ for easy sound design.
- Waveform Mode: Play portions of
- Easily modify attack and release curves.
- Includes equal power cross-fade looping with separate attack and release sample portions.
- Supports pitch bend and fine-tuning.

#### Extras
- Disable antialiasing for a gritty LoFi effect.
- Record samples directly into the sampler.
- Store samples directly in plugin state to remove dependencies (can be disabled).
- Auto-tune to A440 (experimental, works best on simple sounds).

<hr /><em>This discussion was created from the release <a href='https://github.com/BOBONA/Just-a-Sample/releases/tag/'>Just a Sample v1</a>.</em>

**Comment by carmencicero** [comment]
for win11 do i want the just.a.sample.win.zip or the VST3.WIN.zip?

**Comment by BOBONA** [comment]
Just.a.Sample.Win is the installer

**Comment by Dorian-Scale** [comment]
Hello! I'm on windows 10. I have installed THE SAMPLER, it apears in mixcraft vsti 3 browser, but when I open it, this message apears : 
![THE SAMPLER information message](https://github.com/user-attachments/assets/f60162e3-a2cb-4045-b709-f144251cb312)
What's wrong?

**Comment by BOBONA** [comment]
Um, my plugin is not called THE SAMPLER. It’s called Just a Sample. Double
check what you’re opening.

**Comment by Dorian-Scale** [comment]
Oh, you're right. I'm very sorry. Accept my apologies.I checked, and effectively, no problem. Thant you so much for your speed answer and again, i'm sorry. Your product is top! My best wishes.

    Le vendredi 7 février 2025 à 15:04:02 UTC+1, Binyamin Friedman ***@***.***> a écrit :  
 
 

Um, my plugin is not called THE SAMPLER. It’s called Just a Sample. Double
check what you’re opening.
